uctMaths/competition/compadmin.py

```diff
@@ -630,6 +634,51 @@ def export_awards(request):
     output_workbook.save(response)
     return response
 
+def makeCertificates(students, assigned_school):
+    
+    certPath = "../../Certificates/"
+
+    if len(students) > 0:
+        awardCerts = {"G": "goldTemplate.docx", "M": "meritTemplate.docx", None: "participationTemplate.docx", "MOX": "meritOxfordTemplate.docx", "OX": "oxfordTemplate"}
+
+        try:
+            os.mkdir(str(students[0].school).replace(" ", "_") + "_Certificates")
+    
+        finally:
+            certs = []
+            filename = str(str(students[0].school).replace(" ", "_") + "_Certificates")
```

Comment:
We should probably sanitize out `..` and `.` to prevent "hacking"

Reply:
How would I do this, would this involve storing the directory in a variable rather than using paths?

Reply:
It's probably find to just `.replace('.', '_').replace('/', '_')`, it doesn't seem super important to keep dots and slashes.